libcudacxx/include/cuda/std/__type_traits/all_of.h

```diff
@@ -20,22 +20,22 @@
 #  pragma system_header
 #endif // no system header
 
-#include "../__concepts/__concept_macros.h"
-#include "../__type_traits/is_same.h" // IWYU pragma: keep
+#include <cuda/std/__type_traits/is_same.h>
 
 _LIBCUDACXX_BEGIN_NAMESPACE_STD
 
 #if _CCCL_STD_VER >= 2017
 template <bool... _Preds>
-_LIBCUDACXX_CONCEPT __all_of = (_Preds && ...);
+_LIBCUDACXX_INLINE_VAR constexpr bool __all_of = (_Preds && ...);
 #elif _CCCL_STD_VER >= 2014
 template <bool... _Preds>
 struct __all_of_helper;
```

Comment:
nit: this is also used in the implementation of `__none_of`. maybe a name like `__bools` would be better?

Reply:
I dont really like that either and I was unable to find a better name thinking about it